app/assets/javascripts/reducers/uploads.js

```diff
@@ -87,6 +87,20 @@ export default function uploads(state = initialState, action) {
         uploadMetadata: action.data,
       };
     }
+    case SET_UPLOAD_PAGEVIEWS: {
+      const averageViews = [];
+      _.forEach(action.data, fileView => {
+        let fileViews = 0;
+        _.forEach(fileView.items, article => {
+          fileViews += _.get(article, 'views');
+        });
+        averageViews.push(Math.round((fileViews / 60) * 10) / 10);
```

Comment:
It's not guaranteed that you'll receive 60 days worth of results from a pageviews query. If a page has only existed for a short time, the average views should be based on only the number of days available.